src/EA.Weee.RequestHandlers/AatfReturn/Internal/EditAatfDetailsRequestHandler.cs

```diff
@@ -39,12 +48,18 @@ public async Task<bool> HandleAsync(EditAatfDetails message)
                 message.Data.ApprovalNumber,
                 Enumeration.FromValue<Domain.AatfReturn.AatfStatus>(message.Data.AatfStatus.Value),
                 existingAatf.Operator,
-                siteAddress,
+                updatedAddress,
```

Comment:
Can we add some unit tests around updating the address, i.e. calling the mapper and the update data access call